perllib/FixMyStreet/App/Controller/Form.pm

```diff
@@ -87,10 +87,15 @@ sub form : Private {
 
     $form->process unless $form->processed;
 
-    # If we have sent a confirmation email, that function will have
-    # set a template that we need to show
-    $c->stash->{template} = $c->stash->{override_template} || $form->template || $self->index_template
-        unless $c->stash->{sent_confirmation_message};
+    # If the form has the already_submitted_error flag set, show the already_submitted template
+    if ($form->can('already_submitted_error') && $form->already_submitted_error) {
```

Comment:
Do we need the can part here? I think every form used here is a Wizard subclass, so would be good to know we always have it (and put it in if there's somewhere added that lacks it).